protocol/protocol.proto

```diff
@@ -627,11 +627,22 @@ message BlockchainTransaction {
         Event event = 1; // event emitted by the tipping facet
         bytes toUserAddress = 2; // user that received funds
     }
+
+    message TokenTransfer {
+        bytes address = 1;
+        string amount = 2; // uint64 isn't big enough
+        bytes sender = 3;
+        bytes message_id = 4;
+        bytes channel_id = 5;
+        bool is_buy = 6;
+    }
+
     // required fields
     BlockchainTransactionReceipt receipt = 1;
     // optional metadata to be verified by the node
     oneof content {
         Tip tip = 101;
+        TokenTransfer token_transfer = 102;
```

Comment:
heads up i'm adding a new space review type, the merge is going to suck, but i'm numbering mine 103 so you don't have to change this.

Reply:
code is here: https://github.com/towns-protocol/towns/pull/2395